modules/gapi/src/api/gframe.cpp

```diff
@@ -34,7 +29,19 @@ const cv::GOrigin& cv::GFrame::priv() const {
 }
 
 namespace cv {
-std::ostream& operator<<(std::ostream& os, const cv::GFrameDesc &) {
+
+bool GFrameDesc::operator== (const GFrameDesc &rhs) const {
+    return fmt == rhs.fmt && size == rhs.size;
+}
+
+std::ostream& operator<<(std::ostream& os, const cv::GFrameDesc &d) {
```

Comment:
Didn't find `operator>>`. Will it be defined by user?

Reply:
It must be somewhere there

Reply:
https://github.com/opencv/opencv/pull/18494/files#diff-a92ca3714491807461fbd9ae39362adeR564

Reply:
It is there isn't it?